app/controllers/downvotes_controller.rb

```diff
@@ -0,0 +1,29 @@
+class DownvotesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @vote = current_user.votes.new
```

Comment:
so this controller action never `renders` any view templates. So there is really no reason for it to ever set any instance variables.